### PR TITLE
New repository ci-operator configuration for openshift/run-once-duration-override-operator

### PR DESCRIPTION
**Pull request by ingvagabund**
Part of new component github onboarding: https://issues.redhat.com/browse/WRKLDS-639

### CODE REVIEW

File: ci-operator/config/openshift/run-once-duration-override-operator/openshift-run-once-duration-override-operator-main.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+tag_specification:
+  name: "4.13"
+  namespace: ocp
+tests:
+- as: verify
+  commands: make verify-gofmt
+  container:
+    from: src
+- as: unit
+  commands: make test-unit
```
Comment (ardaguclu) [inline ci-operator/config/openshift/run-once-duration-override-operator/openshift-run-once-duration-override-operator-main.yaml:37]: Should these commands be defined in run-once-duration-operator makefile?

Comment (ingvagabund) [reply ci-operator/config/openshift/run-once-duration-override-operator/openshift-run-once-duration-override-operator-main.yaml:37]: They are. Defined through the build-machinery-go.
```sh
$ make help
$ make help
The following make targets are available:
all
build
clean
clean-binaries
clean-controller-gen
clean-yaml-patch
clean-yq
ensure-controller-gen
ensure-imagebuilder
ensure-yaml-patch
ensure-yq
generate
generate-clients
help
image-runoncedurationoverride-operator
images
Makefile
manifests
regen-crd
test
test-e2e
test-unit
update
update-codegen
update-codegen-crds
update-deps-overrides
update-generated
update-gofmt
verify
verify-codegen
verify-codegen-crds
verify-deps
verify-generated
verify-gofmt
verify-golang-versions
verify-golint
verify-govet
```

Comment (ardaguclu) [reply ci-operator/config/openshift/run-once-duration-override-operator/openshift-run-once-duration-override-operator-main.yaml:37]: Learned something new, thanks :)

**Comment by ardaguclu** [comment]
I'm going to investigate why operator_test failed in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/35868/rehearse-35868-pull-ci-openshift-run-once-duration-override-operator-main-e2e-aws-operator/1620410747752288256

**Comment by ingvagabund** [comment]
Addressed through https://github.com/openshift/run-once-duration-override-operator/pull/5

**Pull request merged**